auction-server/src/auction.rs

```diff
@@ -1758,6 +1762,8 @@ impl ChainStore for ChainStoreSvm {
             tracing::warn!(tx_hash = ?tx_hash, bids = ?bids, "multiple bids found for transaction hash");
         }
 
+
+        //TODO: this can be optimized out if triggered by websocket events
```

Comment:
mmm I feel like the `get_signature_status_with_commitment` method is more reliable